bokeh/io/export.py

```diff
@@ -149,8 +149,28 @@ def export_svgs(obj, filename=None, height=None, width=None, webdriver=None):
 # Dev API
 #-----------------------------------------------------------------------------
 
+
 def get_screenshot_as_png(obj, driver=None, **kwargs):
-    '''
+    ''' Get a screenshot of a LayoutDOM object.
+
+    Args:
+        obj (LayoutDOM or Document) : a Layout (Row/Column), Plot or Widget
+            object or Document to export.
+
+        driver (selenium.webdriver) : a selenium webdriver instance to use
+            to export the image.
+
+    Returns:
+        cropped_image (PIL.Image.Image) : a pillow image loaded from PNG.
+
+    .. warning::
+        Responsive sizing_modes may generate layouts with unexpected size and
+        aspect ratios. It is recommended to use the default ``fixed`` sizing mode.
+
+    .. warning::
+        Glyphs that are rendered via webgl won't be included in the generated PNG.
```

Comment:
This depends if the driver supports webgl. This will definitively work with chromedriver.

Reply:
Sure. This section is an exact copy of the docstring from `export_png`, so the change to chromedriver will also affect that. Feel free to use this PR to add the changes to the other docstring, or to copy the docstring found here into a different commit when the internals are updated.

Reply:
Chrome is already an optional option that can be used. Could you update both doc strings? Basically to the effect that webgl may not render with some webdrivers such as phantom JS.

Reply:
On a second thought, this warning is unnecessary. bokehjs falls back to canvas renderer when webgl isn't available or fails for some reason, so all glyphs will be always present in the output.